app/models/spree/adjustment.rb

```diff
@@ -64,7 +64,7 @@ class Adjustment < ActiveRecord::Base
       end
     end
 
-    scope :tax, -> { where(originator_type: 'Spree::TaxRate', adjustable_type: 'Spree::Order') }
+    scope :tax, -> { where(originator_type: 'Spree::TaxRate') }
```

Comment:
Interesting. Australian production:
```
Spree::Adjustment.tax.count
=> 0

Spree::Adjustment.where(originator_type: 'Spree::TaxRate').uniq.pluck(:adjustable_type)
=> ["Spree::LineItem"]
```

So this scope would suddenly return a lot more. Let's look at the usage and impact there.

```
$ gg '\.tax\>'
app/controllers/spree/paypal_controller.rb:      tax_adjustments = order.adjustments.tax.additional
app/controllers/spree/paypal_controller.rb:      tax_adjustments = current_order.adjustments.tax.additional
app/models/spree/order.rb:      adjustments.tax.sum(:amount)
app/models/spree/tax_rate.rb:      order.adjustments.tax.destroy_all
app/models/spree/tax_rate.rb:      # TaxRate adjustments (order.adjustments.tax)
app/models/spree/tax_rate.rb:      (order.adjustments.tax + order.price_adjustments).each do |adjustment|
app/views/producer_mailer/order_cycle_report.html.haml:          %td.tax.text-right
app/views/spree/orders/_form.html.haml:              %span.order-total.tax-total= display_checkout_tax_total(@order)
spec/features/consumer/shopping/cart_spec.rb:        expect(page).to have_selector '.tax-total', text: '11.00' # 10 + 1
spec/mailers/producer_mailer_spec.rb:      .to have_selector("td.tax", text: "$2.73")
spec/models/spree/adjustment_spec.rb:                expect(order.adjustments.tax.first.amount).to eq(5.0)
spec/models/spree/adjustment_spec.rb:                expect(order.adjustments.tax.first.amount).to eq(5.0)
spec/models/spree/adjustment_spec.rb:                expect(order.adjustments.tax.first.amount).to eq(2.0)
spec/models/spree/adjustment_spec.rb:                expect(order.adjustments.tax.first.amount).to eq(12.0)
spec/models/spree/adjustment_spec.rb:            expect(order.all_adjustments.tax.count).to eq 1
spec/models/spree/adjustment_spec.rb:            expect(order.all_adjustments.tax.first.included).to be true
spec/models/spree/adjustment_spec.rb:            expect(order.all_adjustments.tax.count).to eq 1
spec/models/spree/adjustment_spec.rb:            expect(order.all_adjustments.tax.first.included).to be false
```

All these calls are like `order.adjustments.tax` in which case the adjustable_type is already included and this change doesn't have an effect. :heavy_check_mark:

Reply:
Yes, it's related to the slightly mad state we have at the moment, where for instances that use inclusive tax (EU and AUS) the only tax adjustments that we create are on line_items, but for instances that use additional tax (US and CA) the only tax adjustments we create are on orders...